src/pages/Home/index.js

```diff
@@ -26,26 +45,62 @@ const Content = () => {
       <Text size={fontSize} color="#FFFFFF">
         gaming status playing Hack Shack Attack!
       </Text>
-      <Text size={fontSize} color="#FFFFFF">
-        Check out this week in the{' '}
-        <Anchor
-          target="_blank"
-          rel="noopener noreferrer"
-          href="https://vimeo.com/429478014"
-          label="Hack Shack Video"
-          color="teal!"
-        />
-      </Text>
     </TextWrapper>
   );
 };
 
 const Home = () => {
-  // const size = useContext(ResponsiveContext);
+  const size = useContext(ResponsiveContext);
+  const [open, setOpen] = useState();
+  const onOpen = () => setOpen(true);
+  const onClose = () => setOpen(undefined);
 
   return (
     <Layout background="/img/BackgroundImages/hack-shack-home-background.png">
       <Box height="100%" width="100%">
+        {open && (
+          <StyledLayer
+            full
+            animation="fadeIn"
+            onClickOutside={onClose}
+            onEsc={onClose}
+          >
+            <Box alignSelf="center" width="960px" pad={{ top: 'large' }}>
+              <Button
+                alignSelf="end"
+                label={
+                  <Text weight="normal" color="white" size="xlarge">
+                    Close
+                  </Text>
+                }
+                reverse
+                icon={<Close size="medium" />}
+                onClick={onClose}
+              />
+            </Box>
+            <Box alignSelf="center">
+              <Video controls="over" autoPlay fit="cover">
+                <source
+                  key="video"
+                  src={
+                    size !== 'small'
+                      ? 'https://player.vimeo.com/external/430776646.sd.mp4?s=658b900014c931a8ace4fee90658d61fa5b7a133&profile_id=165&download=1'
```

Comment:
Should we consider using 1080p video?
![image](https://user-images.githubusercontent.com/12749370/85186979-c979dc80-b261-11ea-88c2-075ad0b98be1.png)

Reply:
for full size ? sure